nerdlandbot/commands/space_launches.py

```diff
@@ -0,0 +1,117 @@
+import aiohttp
+import asyncio
+import json
+import os
+import discord
+from datetime import datetime
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+
+from nerdlandbot.helpers.constants import \
+                                        THE_SPACE_DEVS_BASE_URL, THE_SPACE_DEVS_VERSION, THE_SPACE_DEVS_LIMIT_TO_10_RESULTS, \
+                                        THE_SPACE_DEVS_UPCOMING_LAUNCH_RESOURCE, \
+                                        THE_SPACE_DEVS_LOCAL_TEST_SERVER_BASE_URL, \
+                                        THE_SPACE_DEVS_LOCAL_CACHE_FOLDER, THE_SPACE_DEVS_LOCAL_CACHE_SPACE_LAUNCHES_FILE, \
+                                        THE_SPACE_DEVS_HOME_URL, THE_SPACE_DEVS_TIMESTAMP_FORMAT
+
+class SpaceDevs (commands.Cog, name='The space devs'):
+    def __init__(self,bot:commands.Bot):
+        self.bot = bot
+        self.ctx = None
```

Comment:
I don't think this line (and line 25) are necessary. You already use `ctx.send` in some places (eg line 41), and you can just use that everywhere instead of `self.ctx.send`

Reply:
There were some ctx.send statements in other functions, but I refactored it so all ctx.sends are in the same function.